PL2/PL2-shared/src/test/java/nl/tudelft/pl2016gr2/dao/AnnotationDAOTest.java

```diff
@@ -0,0 +1,67 @@
+package nl.tudelft.pl2016gr2.dao;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertSame;
+import static org.junit.Assert.assertTrue;
+
+import nl.tudelft.pl2016gr2.model.Annotation;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.stream.Stream;
+
+@SuppressWarnings({"MemberName", "AbbreviationAsWordInName"})
+public class AnnotationDAOTest {
+
+  AnnotationDAO dao;
+  Annotation a1 = new Annotation();
+  Annotation a2 = new Annotation();
+  Annotation a3 = new Annotation();
+  Annotation a4 = new Annotation();
+
+  /**
+   * Setups up the {@link #dao} with the above test annotations.
```

Comment:
`Setups up -> Sets up`

Also, a hyperlink to an object that has no JavaDoc is pointless. use `<code></code>` instead, or add javadoc.